.cirrus.yml

```diff
@@ -226,11 +226,28 @@ publish_images_task:
             sha256sum * > shasums
             popd
 
+            if [[ $CIRRUS_TAG == *-rc* ]]; then
+                RC="--prerelease"
+            else
+                # check if this version should not be marked latest
+                prevrelease=$(curl --retry 3 --silent -m 10 --connect-timeout 5 "https://api.github.com/repos/containers/podman-machine-os/releases/latest")
+                prevvers=$(echo "$prevrelease" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/' | sed -e "s/^v//")
```

Comment:
the payload is just json, seems a lot safer to use `jq -r ".tag_name"` to get the tag name over grep and sed.

Also github started heavy rate limits on unauthenticated requests. I think we can use gh auth login first and then
```
gh api repos/containers/podman-machine-os/releases/latest --jq '.tag_name'
``` 
which should make the same request as curl and we don't need to filter the output ourselves